runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -3301,13 +3301,11 @@ J9::ARM64::TreeEvaluator::evaluateNULLCHKWithPossibleResolve(TR::Node *node, boo
       // pattern match the sequence under the l2a
       // NULLCHK        NULLCHK                     <- node
       //    aloadi f      l2a
-      //       aload O       ladd
-      //                       lshl
-      //                          i2l
-      //                            iloadi/irdbari f <- n
-      //                               aload O        <- reference
-      //                          iconst shftKonst
-      //                       lconst HB
+      //       aload O      lshl
+      //                       iu2l
+      //                          iloadi/irdbari f <- n
+      //                             aload O        <- reference
+      //                       iconst shftKonst
```

Comment:
I updated the compressed sequence tree in the comment to be consistent with the current IL.
We do not have `ladd` anymore since heapbase has been eliminated in https://github.com/eclipse/openj9/pull/11688